portfolio/src/main/webapp/script.js

```diff
@@ -55,8 +55,147 @@ function deleteTask(task) {
 }
 
 /** Creates a map and adds it to the page. */
+// function createMap() {
```

Comment:
you can delete these commented out lines :)

Reply:
I removed them in info window pr :)